test.js

```diff
@@ -4,5 +4,13 @@ var fileUrl = require('./');
 
 it('should convert path to file url', function () {
 	assert(/file:\/\/\/.*\/test\.jpg/.test(fileUrl('test.jpg')));
-	assert.strictEqual(fileUrl('/Users/sindresorhus/dev/te^st.jpg'), 'file:///Users/sindresorhus/dev/te%5Est.jpg');
+	if (process.platform === 'win32') {
```

Comment:
It doesn't matter what drive the user runs this on as I pass an absolute path that already contains a drive letter. In fact, I just ran the tests on my `D:` drive and they pass just fine. It is just about passing a Windows-compatible path.